src/sample/stuDashController.java

```diff
@@ -82,19 +108,221 @@ public void saveInfo() {
     nric.setStyle("-fx-border-color: default");
     cgpa.setStyle("-fx-border-color: default");
 
+    try{
+      File stuinfoFile = new File("data/Student Dashboard/"+ id +".txt");
+      stuinfoFile.createNewFile();
+      BufferedWriter writer=new BufferedWriter(new FileWriter("data/Student Dashboard/"+ id +".txt"));
+      writer.write(name.getText());
+      writer.write("\n" + nric.getText());
+      writer.write("\n" + id);
+      writer.write("\n" + acd_status.getSelectionModel().getSelectedItem());
+      writer.write("\n" + sem_reg.getSelectionModel().getSelectedItem());
+      writer.write("\n" + cgpa.getText());
+      writer.write("\n" + year.getSelectionModel().getSelectedItem());
+      writer.write("\n" + school.getSelectionModel().getSelectedItem());
+      writer.write("\n" + campus.getSelectionModel().getSelectedItem());
+      writer.write("\n" + programme.getSelectionModel().getSelectedItem());
+      writer.write("\n" + major.getSelectionModel().getSelectedItem());
+      writer.write("\n" + minor.getSelectionModel().getSelectedItem());
+      writer.close();
+      
+    }catch(IOException e)
+    {
+      System.out.println("An error occured.");
+    }
+
+
+  }
+  
+  
+  public void displayName() 
```

Comment:
can you make all the display... function to be the same function, they shows similar functionality

Reply:
ya i know they have the similar functionality, but cannot because i think this way is much more feasible and it shows output as I want, if wanna make changes I need to figure out the way again which is time-consuming I believe

Reply:
but before this i tried to combine all the display function into a function called defaultInfo() function, but when I log in it does not immediately appear the text, it will only appear after I switch the scene